front/src/applications/stdcm/hooks/useStdcmForm.ts

```diff
@@ -38,6 +38,7 @@ const useStdcmForm = (): StdcmSimulationInputs => {
       pathSteps,
       departureDate: originArrival?.arrivalDate,
       departureTime: originArrival?.arrivalTime,
+      tolerances: origin.tolerances,
```

Comment:
What if the tolerances are on the destination ? 
Moreover, with this data structure, you will loose a piece of information: on which extremity the tolerances are defined :/

Reply:
Done !